MrTimms/MrTimms-Phx.md

```diff
@@ -0,0 +1,7 @@
+# Phoenix
+
+
+### Restaurants
+
+
+### Thigns to Do
```

Comment:
I have only driven through Phoenix but never stayed. Looking forward to seeing what things there are to do.